ocaml/typing/persistent_env.ml

```diff
@@ -46,17 +46,9 @@ let error err = raise (Error err)
 module Persistent_signature = struct
   type t =
     { filename : string;
-<<<<<<< HEAD
       cmi : Cmi_format.cmi_infos_lazy;
       visibility : Load_path.visibility }
-||||||| 121bedcfd2
-      cmi : Cmi_format.cmi_infos }
-=======
-      cmi : Cmi_format.cmi_infos;
-      visibility : Load_path.visibility }
->>>>>>> 5.2.0
 
-<<<<<<< HEAD
   let load = ref (fun ~allow_hidden ~unit_name ->
     let unit_name = CU.Name.to_string unit_name in
     match Load_path.find_uncap_with_visibility (unit_name ^ ".cmi") with
```

Comment:
I think it was missed in the merge of `Load_path` that we should take the new name for this function (`find_normalized_with_visibility`).

Reply:
I've done that renaming within `Load_path` (of `uncap`-like names to `normalize`-like names). This should be purely a no-op: `normalize`-ing something is `String.lowercase_ascii`.

Reply:
I'll let you resolve @ccasin if the last commit looks fine.

Reply:
Last commit looks good, thanks for doing the rename!